src/platform/macos/view.rs

```diff
@@ -403,6 +403,44 @@ fn get_characters(event: id) -> Option<String> {
     }
 }
 
+fn get_characters_ignoring_modifiers(event: id) -> Option<String> {
+    unsafe {
+        let characters: id = msg_send![event, charactersIgnoringModifiers];
+        let slice = slice::from_raw_parts(
+            characters.UTF8String() as *const c_uchar,
+            characters.len(),
+        );
+        let string = str::from_utf8_unchecked(slice);
+        Some(string.to_owned())
+    }
+}
+
+// Translate macos scancode into the corresponding windows key code.
+fn to_translated_macos_scancode(event: id) -> c_ushort {
+    // Cmd switches Roman letters for Dvorak-QWERTY layout, so try modified characters first.
+    let characters = get_characters(event);
+    if let Some(chars) = characters {
+        let code = chars.chars().next().map_or(0, |c| windows_key_code_for_char_code(c).unwrap_or(0));
+        if code != 0 {
+            return code;
+        }
+    }
+
+    // Translate characters that weren't caught in the above operation.
```

Comment:
Are there any examples of when this is triggered?

Reply:
Yes.  Mostly keys such as backspace and enter.  I'll elaborate with these in the comment.